### DISCUSSION TITLE
Find a Scanner Button

### DISCUSSION THREAD

**Discussion by hugodg** [wazuh/wazuh#13397] [General]
Hello good afternoon
     I'm starting to work with Wazuh, but I'm having a great difficulty that would be "Find a Scanner Button" without having to wait a certain time programmed, could you let me know if there is some kind of "Scanner Button"??
![Captura de tela de 2022-05-06 16-31-24](https://user-images.githubusercontent.com/2335368/167205349-e3753318-f52b-4cc3-af4d-8ab3ac547711.png)

**Comment by MiguelazoDS** [comment]
Hello @hugodg, 

What feature are you referring to? If you are thinking of Vulnerability detector or Syscollector, those modules currently perform scans only by an interval. Although there's a work in progress to execute scans on demand. 

This is the issue of this implementation https://github.com/wazuh/wazuh/issues/9220

Regards!
